code/style.css

```diff
@@ -1,4 +1,268 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  background: #8c8d9240;
+  color: #404040;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  box-sizing: border-box;
+  margin: 0;
 }
+
+/* hover effect nav bar */
+
+a:link {
+  color: black;
+}
+
+/* visited link */
+a:visited {
+  color: #404040;
+}
+
+/* mouse over link */
+a:hover {
+  color: #6e7e85;
+}
+
+/* selected link */
+a:active {
+  color: #d09a99;
+} 
+
+/* header */
+
+h1, h2{
+  font: 70px;
+  text-align: center
+}
+
+.top-header{
+  display: flex;
+  flex-direction: column;
+  justify-content:space-evenly;
+  align-items: center;
+  height: 360px;
+  border-radius: 1rem;
+  padding: 25px;
+  background: linear-gradient(
+      45deg,
+      #6e7e85,
+      #8aa2a9,
+      #b7cece,
+      #d5b9b2,
+      #ddd5d0,
+      #c3c3c3);
+
+  background-size: 500%;
+  animation-name: gradient;
+  animation-duration: 10s;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+}
+@keyframes gradient{
+  from {
+      background-position: 0%;
+  }
+  to {
+      background-position: 100%;
+  }
+  }
+
+/* nav bar + logo */
+
+.logo-pic {
+  height: 200px;
+  width: 200px;
+  align-items: center;
+}
+
+.nav-bar{
+  width: 100%;
+  display: flex;
+  text-align: right;
+  justify-content:space-evenly;
+  font-size: 24px;
+  border-radius: 1rem;
+}
+
+/* big size news */
+
+.big-news {
+  display: grid;
+  grid-template-columns:1fr;
+  grid-gap: 2px;
+  border-radius: 1rem;
+  border:  1px solid black;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.big-pic {
+  height: 400px;
+  width: 600px;
+  border-radius: 1rem;
+  box-shadow: 13px 10px;
+  padding-left: 15px;
+}
+
+.imagewrapper {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-top: 35px;
+}
+
+.grid-1{
+  margin-top: 20px;
+  padding: 25px;
+  align-items: right;
+  text-align: center;
+}
+
+/* medium size news */
+
+.small-news{
+  display: grid;
+  grid-template-columns:1fr;
+  grid-gap: 5px;
+  margin-top: 5px;
+  text-align: center;
+}
+
+.grid-2, .grid-7 {
+  display: flex;
+  border:  1px solid black;
+  border-radius: 1rem;
+  grid-column: span 1;
+  align-items: center;
+  justify-content: center;
+  padding-left: 15px;
+  padding-top: 3px;
+}
+
+.small-pic{
+  max-width: 300px;
+  max-height: 300px;
+  border-radius: 1rem;
+  box-shadow: 5px 10px;
+  margin-right: 10px;
+}
+
+/* smaller size news */
+
+.grid-3, .grid-4, .grid-5, .grid-6 {
+  display: flex;
+  flex-flow: column;
+  border:  1px solid black;
+  border-radius: 1rem;
+  padding: 2rem;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+}
+
+.little-pic{
+  height: 250px;
+  width: 250px;
+  border-radius: 1rem;
+  box-shadow: 5px 10px;
+}
+
+.sub-text {
+  margin-top: 0;
+}
+
+/* footer */
```

Comment:
The footer isn't in the bottom of the page. Maybe it works to write: 
margin: 0px;
or
margin-bottom: 0px;

Or maybe it is because the footer lies inside the header in the HTML?